v1.0.0.yaml

```
@@ -2809,131 +2722,88 @@ paths:
-                  example: https://www.example.com/rest-of-the-image-path.jpg.
-                fileName:
-                  type: string
-                  description: |
-                    The name with which the file has to be uploaded.
-
-                    The file name can contain:
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          description: >
+            Authorization is different for server-side vs client-side file upload.
 
-                    - Alphanumeric Characters: `a-z`, `A-Z`, `0-9` (including unicode letters, marks, and numerals in other languages)
-                    - Special Characters: `.` and `-`
-                    Any other character including space will be replaced by `_`
-                useUniqueFileName:
-                  type: boolean
-                  description: |
-                    Whether to use a unique filename for this file or not.
+            ### Authorization for server-side file upload
 
-                    - Accepts `true` or `false`.
-                    - If set `true`, ImageKit.io will add a unique suffix to the filename parameter to get a unique filename.
-                    - If set `false`, then the image is uploaded with the provided filename parameter, and any existing file with the same name is replaced.
+            Basic HTTP authentication. Provide base64 encoded value of the string `your_private_api_key:`.
 
-                    Default value - `true`
-                tags:
-                  type: string
-                  description: |
-                    Set the tags while uploading the file.
-
-                    - Comma-separated value of tags in the format `tag1,tag2,tag3`.
-                    - The maximum length of all characters should not exceed 500.
-                    - `%` is not allowed.
-                    - If this field is not specified and the file is overwritten then the tags will be removed.
-                  example: t-shirt,round-neck,men
-                folder:
-                  type: string
-                  description: |
-                    The folder path (e.g. `/images/folder/`) in which the image has to be uploaded. If the folder(s) didn't exist before, a new folder(s) is created. The nesting of folders can be at most 50 levels deep.
+        
+            Notice the colon (:) after the private key. It is required otherwise, authentication will fail. The format is `username:password`. `username` is your private key and `password` is an empty string.
 
-                    The folder name can contain:
+        
+            If you encode `your_private_api_key:` using base64, you will get `eW91cl9wcml2YXRlX2FwaV9rZXk6`.
```

This example is a bit confusing. Can we just encode a fake key instead? 
```suggestion
            If your private key is `private_rGAPQJbhBx`, encode `private_rGAPQJbhBx:` using base64. The encoded value will be `cHJpdmF0ZV9yR0FQUUpiaEJ4Og==` in this case.
```

---

pick from stripe maybe - https://docs.stripe.com/api/authentication